io-engine/src/bdev/nexus/nexus_bdev_rebuild.rs

```diff
@@ -151,6 +153,31 @@ impl<'n> Nexus<'n> {
         src_child_uri: &str,
         dst_child_uri: &str,
     ) -> Result<(), Error> {
+        let verify_mode = match std::env::var("NEXUS_REBUILD_VERIFY")
```

Comment:
Could this be a CLI argument?
Ex: sudo io-engine --rebuild-verify-ios

Reply:
We would end up with too many CLI args. Also, we were planned to make this a parameter of rebuild start / child online API.

Reply:
We can have a separate struct for Testing cli arguments, then it's clear any "user" would ignore it but it will make it visible for testing what's available.